### ISSUE TITLE
API docs should use `https` instead of `http`

### ISSUE THREAD

**Issue by xiongtx** [clojure-grimoire/grimoire#263]
The HTTP API is no longer available, even though the [docs still refer to it](https://www.conj.io/api).

```
$ curl -X GET -H 'Content-Type:text/plain' http://www.conj.io/store/v1/org.clojure/clojure/1.8.0/clj/clojure.core/conj

<html>
<head><title>301 Moved Permanently</title></head>
<body bgcolor="white">
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx/1.10.0 (Ubuntu)</center>
</body>
</html>

```

**Comment by arrdem** [comment]
Thanks! I'll patch this in a second.

**Comment by xiongtx** [comment]
Holy shit that was fast :scream:.

**Comment by arrdem** [comment]
I happened to have just finished typing up jaunt-lang/jaunt#157. I do have a life :stuck_out_tongue:

**Comment by xiongtx** [comment]
*Sets up PD for all GitHub notifications*

**Comment by arrdem** [comment]
Please don't

**Comment by arrdem** [comment]
Fixed in b320c7c15c1d5032f8349800531650dfcf4ccc04
